fog/view/server/src/lib.rs

```diff
@@ -13,5 +13,6 @@ pub mod sharding_strategy;
 mod block_tracker;
 mod counters;
 mod db_fetcher;
+mod fog_view_router_admin_service;
```

Comment:
So, the full path to this thing is:

`mc_fog_view_server::fog_view_router_admin_service;`

Can this just be `router_admin_service`?

Reply:
Yes, done.